### PR TITLE
Tweak messaging on load

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: R/project.R [hunk 1]
```diff
@@ -270,24 +270,24 @@ renv_project_synchronized_check <- function(project = NULL, lockfile = NULL) {
...
+      "- One or more packages recorded in the lockfile are not installed.",
+      "- Use `renv::status()` for more details."
     )
     writef(msg)
     return(FALSE)
   }
 
   # otherwise, use status to detect if we're synchronized
-  info <- quietly({
+  info <- local({
```
Comment (hadley) [inline R/project.R:281]: This is really the guts of the PR — previously this didn't create a new scope, so the message about the project being out of sync was never shown.

File: R/load.R [hunk 2]
```diff
@@ -729,15 +730,13 @@ renv_load_quiet <- function() {
...
 
-  if (!renv_load_quiet()) {
-    renv_load_report_project(project)
-    renv_load_report_python(project)
-  }
+  renv_load_check(project)
+  renv_load_report_project(project)
+  renv_load_report_python(project)
 
   if (interactive()) {
```
Comment (hadley) [inline R/load.R:741]: Is it really the case we should only do this reports interactively? If so, should `renv_snapshot_auto_update()` also only be done interactively? The current behaviour seems inconsistent to me.

File: R/load.R [hunk 3]
```diff
@@ -91,9 +91,10 @@ load <- function(project = NULL, quiet = FALSE) {
   if (switch)
     return(renv_load_switch(project))
 
-  if (quiet)
+  if (quiet || renv_session_quiet())
     renv_scope_options(renv.verbose = FALSE)
 
+  writef(header("Loading renv [%s]", renv_metadata_version_friendly()))
```
Comment (hadley) [inline R/load.R:97]: IMO a header makes sense here because it gets bullet updates.

File: R/load.R [hunk 4]
```diff
@@ -757,11 +756,11 @@ renv_load_report_project <- function(project) {
   version <- renv_metadata_version_friendly()
 
   if (length(profile)) {
-    fmt <- "* (%s) Project '%s' loaded. [renv %s]"
-    writef(fmt, profile, renv_path_aliased(project), version)
+    fmt <- "- Project '%s' loaded with %s profile."
```
Comment (kevinushey) [inline R/load.R:759]: Do we want to make a decision around whether we use `*` prefixes or `-` prefixes for messages like these? Or maybe we should start wrapping these into internal calls to cli-like functions like `inform()` so we can easily adapt in the future?

**Review by kevinushey — APPROVED** [review]

File: R/load.R (hunk 2 above)
Comment (kevinushey) [reply R/load.R:741]: The intention was for automatic snapshots to only happen in interactive sessions, so I'm happy to make things more consistent here.

File: R/load.R (hunk 4 above)
Comment (hadley) [reply R/load.R:757]: I thought we were moving towards `-` everywhere?

Comment (kevinushey) [reply R/load.R:757]: Did we make that decision explicitly? I'm happy to do it but can't recall if we consciously chose to.

Comment (hadley) [reply R/load.R:757]: I don't think we discussed it but I think I assumed based on your recent PRs 😄

**Pull request merged**